ocf_datapipes/training/windnet.py

```diff
@@ -268,7 +221,9 @@ def construct_sliced_data_pipeline(
         fork_keys.update(set(f"nwp/{k}" for k in datapipes_dict["nwp"].keys()))
 
     # We don't need somes keys even if they are in the data dictionary
-    fork_keys = fork_keys - set(["topo", "nwp", "sensor", "hrv", "pv_future", "pv"])
+    fork_keys = fork_keys - set(
```

Comment:
@dfulu Just to I understand, to use the correct key for the location datapipe, "wind", , should I have it be removed from this set, or in it?

Reply:
What you've done looks right, or at least consistant. This code could probably be clearer, but the `fork_keys` are the keys that we will need a copy of the location datapipe for. It looks like you aren't slicing the wind data by location further below(?) so it shouldn't be in `fork_set`